scripts/qa_crowbarsetup.sh

```diff
@@ -3608,10 +3608,23 @@ function oncontroller_run_tempest
     pushd /var/lib/openstack-tempest-test
     sysctl -e kernel.sysrq=1 net.ipv4.neigh.default.gc_thresh1=0
     local tempestret
+    local blacklist=""
+
+    if iscloudver 9; then
+
+        blacklistfile=/var/lib/openstack-tempest-test/blacklist.txt
+        blacklist="--blacklist-file $blacklistfile"
+        cat -  << EOF
+id-f5dfcc22-45fd-409f-954c-5bd500d7890b
+id-301f5a30-1c6f-4ea0-be1a-91fd28d44354
+id-bdbb5441-9204-419d-a225-b4fdbfb1a1a8
```

Comment:
This one is test_basic_minimum...it's sad we don't have that working

Reply:
agreed, but to be positive: I think its just a few tests that are failing..